### ISSUE TITLE
`Sync isn't working` notification is seen after joining the existing sync chain

### ISSUE THREAD

**Issue by hffvld** [brave/brave-browser#38176]
## Description 

`Sync isn't working` notification is seen after joining the existing sync chain.

## Steps to reproduce 

   1. Launch Brave `1.65.110` or newer
   2. Open `Settings` > Sync > Scan or enter sync code
   3. Scan the QR code or enter the phrase for the existing sync chain
   4. When you see the `Warning` message, tap `Yes` > *Observe the notification*

## Actual result 

`Sync isn't working` notification is shown right after the user joins the existing sync chain by scanning the QR code or entering the phrase (no repro when starting the sync chain on the Android device itself). This notification begins appearing with Brave `1.65.110` which has `Ch124`and not reproducible in Brave `1.64.116` with `Ch123` or older builds.

|1|2|
|:-:|:-:|
| ![1](https://github.com/brave/brave-browser/assets/128532965/5b9e4621-415d-4c21-ab7d-5cc522ee45d4) | ![2](https://github.com/brave/brave-browser/assets/128532965/165c2dcd-08cd-4d25-82a4-6e9b862bb103) |

```
2024-05-08 15:52:40.147  9578-9578  DynamiteModule          com.brave.browser                    W  Local module descriptor class for com.google.android.gms.vision.dynamite.barcode not found.
2024-05-08 15:52:40.156  9578-9578  DynamiteModule          com.brave.browser                    I  Considering local module com.google.android.gms.vision.dynamite.barcode:0 and remote module com.google.android.gms.vision.dynamite.barcode:0
2024-05-08 15:52:40.159  9578-9578  DynamiteModule          com.brave.browser                    I  Considering local module com.google.android.gms.vision.barcode:0 and remote module com.google.android.gms.vision.barcode:241616000
2024-05-08 15:52:40.159  9578-9578  DynamiteModule          com.brave.browser                    I  Selected remote version of com.google.android.gms.vision.barcode, version >= 241616000
2024-05-08 15:52:40.173  9578-9578  CameraBase              com.brave.browser                    I  Connect camera (legacy API) - overrideToPortrait 0, forceSlowJpegMode 0
2024-05-08 15:52:40.174  1069-1069  CameraService           cameraserver                         I  CameraService::connect call (PID 9578 "com.brave.browser", camera ID 0) and Camera API version 1
2024-05-08 15:52:40.175  1069-1069  Camera2ClientBase       cameraserver                         I  Camera 0: Opened. Client: com.brave.browser (PID 9578, UID 11008)
2024-05-08 15:52:40.189  2333-2536  LyricConfi...derService com.google.pixel.camera.services     D  onCameraOpened: cameraId 0 used by packageId com.brave.browser for versionCode 426513024
2024-05-08 15:52:40.250   552-32381 gralloc4                and...graphics.allocator-V1-service  W  Unable to set buffer name SurfaceView[com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity]#31(BLAST Consumer)31: File name too long
2024-05-08 15:52:40.415   552-3368  gralloc4                and...graphics.allocator-V1-service  W  Unable to set buffer name SurfaceView[com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity]#31(BLAST Consumer)31: File name too long
2024-05-08 15:52:40.486   552-32381 gralloc4                and...graphics.allocator-V1-service  W  Unable to set buffer name SurfaceView[com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity]#31(BLAST Consumer)31: File name too long
2024-05-08 15:52:41.369  9578-9578  ResourcesCompat         com.brave.browser                    E  Failed to find font-family tag
2024-05-08 15:52:41.394  1580-3519  CoreBackPreview         system_server                        D  Window{ec50f19 u0 com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity}: Setting back callback OnBackInvokedCallbackInfo{mCallback=android.window.IOnBackInvokedCallback$Stub$Proxy@9737ea, mPriority=-1, mIsAnimationCallback=false}
2024-05-08 15:52:41.416  1580-1884  WindowManager           system_server                        V  Loading animations: layout params pkg=com.brave.browser resId=0x7f15000a
2024-05-08 15:52:41.416  1580-1884  WindowManager           system_server                        V  Loading animations: picked package=com.brave.browser
2024-05-08 15:52:41.427   552-32381 gralloc4                and...graphics.allocator-V1-service  W  Unable to set buffer name SurfaceView[com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity]#31(BLAST Consumer)31: File name too long
2024-05-08 15:52:41.750  9578-9583  m.brave.browser         com.brave.browser                    I  Background concurrent mark compact GC freed 16MB AllocSpace bytes, 54(23MB) LOS objects, 73% free, 34MB/130MB, paused 2.339ms,2.971ms total 137.268ms
2024-05-08 15:52:42.531  1580-3519  ProcessStats            system_server                        W  Tracking association SourceState{1cda0cb com.brave.browser/11008 BTop #193126} whose proc state 2 is better than process ProcessState{11940a8 com.google.android.gms.persistent/10130 pkg=com.google.android.gms} proc state 14 (330 skipped)
2024-05-08 15:52:42.664  9578-9578  WindowOnBackDispatcher  com.brave.browser                    W  sendCancelIfRunning: isInProgress=falsecallback=android.app.Dialog$$ExternalSyntheticLambda2@ab2059f
2024-05-08 15:52:42.664  1580-4452  CoreBackPreview         system_server                        D  Window{ec50f19 u0 com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity}: Setting back callback null
2024-05-08 15:52:42.668  9578-9631  HWUI                    com.brave.browser                    D  endAllActiveAnimators on 0xb400007d9eb36210 (RippleDrawable) with handle 0xb400007c1e7fb5a0
2024-05-08 15:52:42.669  1580-4452  InputManager-JNI        system_server                        W  Input channel object 'ec50f19 com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity (client)' was disposed without first being removed with the input manager!
2024-05-08 15:52:42.670  1580-4452  WindowManager           system_server                        V  Loading animations: layout params pkg=com.brave.browser resId=0x7f15000a
2024-05-08 15:52:42.670  1580-4452  WindowManager           system_server                        V  Loading animations: picked package=com.brave.browser
2024-05-08 15:52:42.677  9578-9578  chromium                com.brave.browser                    W  [WARNING:instance_id_android.cc(99)] Non-zero TTL requested for InstanceID token, while TTLs are not supported by Android Firebase IID API.
2024-05-08 15:52:42.715  1580-3519  ImeTracker              system_server                        I  com.brave.browser:ea0019f0: onRequestHide at ORIGIN_SERVER reason HIDE_SAME_WINDOW_FOCUSED_WITHOUT_EDITOR fromUser false
2024-05-08 15:52:42.716  1580-3519  ImeTracker              system_server                        I  com.brave.browser:ea0019f0: onCancelled at PHASE_SERVER_SHOULD_HIDE
2024-05-08 15:52:42.720  3253-3253  GoogleInpu...hodService com...gle.android.inputmethod.latin  I  GoogleInputMethodService.onStartInput():1954 onStartInput(EditorInfo{EditorInfo{packageName=com.brave.browser, inputType=0, inputTypeString=NULL, enableLearning=false, autoCorrection=false, autoComplete=false, imeOptions=0, privateImeOptions=null, actionName=UNSPECIFIED, actionLabel=null, initialSelStart=-1, initialSelEnd=-1, initialCapsMode=0, label=null, fieldId=0, fieldName=null, extras=null, hintText=null, hintLocales=[]}}, false)
2024-05-08 15:52:42.724  1580-3519  PackageConfigPersister  system_server                        W  App-specific configuration not found for packageName: com.brave.browser and userId: 0
2024-05-08 15:52:42.788  9578-9578  chromium                com.brave.browser                    E  [ERROR:data_type_manager_impl.cc(36)] Bookmarks cryptographer error was encountered: 
2024-05-08 15:52:42.788  9578-9578  chromium                com.brave.browser                    E  [ERROR:data_type_manager_impl.cc(36)] Device Info cryptographer error was encountered: 
2024-05-08 15:52:42.788  9578-9578  chromium                com.brave.browser                    E  [ERROR:data_type_manager_impl.cc(36)] Send Tab To Self cryptographer error was encountered: 
2024-05-08 15:52:43.125  9578-9578  cr_SYNC                 com.brave.browser                    W  No need to load devices for other pages
2024-05-08 15:52:43.138  9578-9578  Camera                  com.brave.browser                    D  app passed NULL surface
2024-05-08 15:52:43.260  1069-1069  Camera2ClientBase       cameraserver                         I  ~Camera2ClientBase: Client object's dtor for Camera Id 0 completed. Client was: com.brave.browser (PID 9578, UID 11008)
2024-05-08 15:52:43.261  9578-9578  cr_SYNC                 com.brave.browser                    I  GetSyncDeviceList devices length is 6
2024-05-08 15:52:43.636  9578-9578  cr_SYNC                 com.brave.browser                    I  GetSyncDeviceList devices length is 6
2024-05-08 15:52:43.880  9578-9578  cr_SYNC                 com.brave.browser                    I  GetSyncDeviceList devices length is 6
2024-05-08 15:53:09.168  2157-2176  WindowManagerShell      com.android.systemui                 V  Transition requested (#1665): android.os.BinderProxy@bd9734d TransitionRequestInfo { type = OPEN, triggerTask = TaskInfo{userId=0 taskId=12074 displayId=0 isRunning=true baseIntent=Intent { act=notifications.NotificationIntentInterceptor.INTENT_ACTION flg=0x188c0000 cmp=com.brave.browser/org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity } baseActivity=ComponentInfo{com.brave.browser/org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity} topActivity=ComponentInfo{com.brave.browser/org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity} origActivity=null realActivity=ComponentInfo{com.brave.browser/org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity} numActivities=1 lastActiveTime=190859316 supportsMultiWindow=true resizeMode=1 isResizeable=true minWidth=-1 minHeight=-1 defaultMinSize=220 token=WCT{android.window.IWindowContainerToken$Stub$Proxy@745be02} topActivityType=1 pictureInPictureParams=null shouldDockBigOverlays=false launchIntoPipHostTaskId=-1 lastParentTaskIdBeforePip=-1 displayCutoutSafeInsets=Rect(0, 136 - 0, 0) topActivityInfo=ActivityInfo{22b313 org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity} launchCookies=[] positionInParent=Point(0, 0) parentTaskId=-1 isFocused=false isVisible=false isVisibleRequested=false isSleeping=false locusId=null displayAreaFeatureId=1 isTopActivityTransparent=true appCompatTaskInfo=AppCompatTaskInfo { topActivityInSizeCompat=false topActivityEligibleForLetterboxEducation= false isLetterboxDoubleTapEnabled= false topActivityEligibleForUserAspectRatioButton= false topActivityBoundsLetterboxed= false isFromLetterboxDoubleTap= false topActivityLetterboxVerticalPosition= -1 topActivityLetterboxHorizontalPosition= -1 topActivityLetterboxWidth=1080 topActivityLetterboxHeight=2400 isUserFullscreenOverrideEnabled=false isSystemFullscreenOverrideEnabled=false cameraCompatControlState=hidden}}, pipTask = null, remoteTransition = RemoteTransition { remoteTransition = com.android.systemui.animation.RemoteAnimationRunnerCompat$1@5083950, appThread = null, debugName = SysUILaunch }, displayChange = null, flags = 0, debugId = 1665 }
2024-05-08 15:53:09.168  1580-3648  ActivityTaskManager     system_server                        I  START u0 {act=notifications.NotificationIntentInterceptor.INTENT_ACTION flg=0x40000 cmp=com.brave.browser/org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity (has extras)} with LAUNCH_MULTIPLE from uid 11008 (realCallingUid=10218) (BAL_ALLOW_VISIBLE_WINDOW) result code=0
2024-05-08 15:53:09.171  9578-9578  ImeTracker              com.brave.browser                    I  com.brave.browser:50829a20: onRequestHide at ORIGIN_CLIENT reason HIDE_SOFT_INPUT fromUser false
2024-05-08 15:53:09.171  1580-3694  ImeTracker              system_server                        I  com.brave.browser:50829a20: onCancelled at PHASE_SERVER_SHOULD_HIDE
2024-05-08 15:53:09.178  9578-9578  cr_OfflineDetector      com.brave.browser                    I  Running updateState mConnectivityDetectorInitialized: true, mTimeWhenLastForegrounded: 190859327, getElapsedTime: 190859327, mTimeWhenLastOfflineNotificationReceived: 0, mTimeWhenLastOnline: 189134250, mTimeWhenAirplaneModeToggledOff: 0 mApplicationState: 1, mIsOfflineLastReportedByConnectivityDetector: false, mInAirplaneMode: false mIsEffectivelyOffline: false
2024-05-08 15:53:09.178  9578-9578  cr_OfflineDetector      com.brave.browser                    I  updateState(): timeSinceLastForeground: 0, timeSinceOfflineNotificationReceived: 190859327, timeSinceLastOnline: 1725077, timeNeededForForeground: 2000, timeNeededForOffline: -190857327 timeSinceAirplaneModeToggledOff: 190859327 timeNeededAfterConnectionChangeFromOnlineToOffline: -1715077 timeNeededAfterConnectionChangeFromAirplaneToOffline: 0
2024-05-08 15:53:09.182  1580-3648  ActivityTaskManager     system_server                        I  START u0 {flg=0x14000000 cmp=com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity} with LAUNCH_MULTIPLE from uid 11008 (BAL_ALLOW_VISIBLE_WINDOW) result code=0
2024-05-08 15:53:09.198  1580-1880  WindowManager           system_server                        V  Sent Transition (#1665) createdAt=05-08 15:53:09.156 via request=TransitionRequestInfo { type = OPEN, triggerTask = TaskInfo{userId=0 taskId=12074 displayId=0 isRunning=true baseIntent=Intent { act=notifications.NotificationIntentInterceptor.INTENT_ACTION flg=0x188c0000 cmp=com.brave.browser/org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity } baseActivity=ComponentInfo{com.brave.browser/org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity} topActivity=ComponentInfo{com.brave.browser/org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity} origActivity=null realActivity=ComponentInfo{com.brave.browser/org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity} numActivities=1 lastActiveTime=190859316 supportsMultiWindow=true resizeMode=1 isResizeable=true minWidth=-1 minHeight=-1 defaultMinSize=220 token=WCT{RemoteToken{44def41 Task{161b16a #12074 type=standard A=11008:com.brave.browser}}} topActivityType=1 pictureInPictureParams=null shouldDockBigOverlays=false launchIntoPipHostTaskId=-1 lastParentTaskIdBeforePip=-1 displayCutoutSafeInsets=Rect(0, 136 - 0, 0) topActivityInfo=ActivityInfo{15adce6 org.chromium.chrome.browser.notifications.NotificationIntentInterceptor$TrampolineActivity} launchCookies=[] positionInParent=Point(0, 0) parentTaskId=-1 isFocused=false isVisible=false isVisibleRequested=false isSleeping=false locusId=null displayAreaFeatureId=1 isTopActivityTransparent=true appCompatTaskInfo=AppCompatTaskInfo { topActivityInSizeCompat=false topActivityEligibleForLetterboxEducation= false isLetterboxDoubleTapEnabled= false topActivityEligibleForUserAspectRatioButton= false topActivityBoundsLetterboxed= false isFromLetterboxDoubleTap= false topActivityLetterboxVerticalPosition= -1 topActivityLetterboxHorizontalPosition= -1 topActivityLetterboxWidth=1080 topActivityLetterboxHeight=2400 isUserFullscreenOverrideEnabled=false isSystemFullscreenOverrideEnabled=false cameraCompatControlState=hidden}}, pipTask = null, remoteTransition = RemoteTransition { remoteTransition = android.window.IRemoteTransition$Stub$Proxy@2e25227, appThread = null, debugName = SysUILaunch }, displayChange = null, flags = 0, debugId = 1665 }
2024-05-08 15:53:09.199  1580-1880  WindowManager           system_server                        V      info={id=1665 t=OPEN f=0x0 trk=0 r=[0@Point(0, 0)] c=[
                                                                                                            {null m=OPEN f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW p=WCT{RemoteToken{d89069d Task{45cb82d #12070 type=standard A=11008:com.brave.browser}}} leash=Surface(name=ActivityRecord{226fd1 u0 com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity t12070})/@0x9ee3c4b sb=Rect(0, 0 - 0, 0) eb=Rect(0, 0 - 1080, 2400) d=-1->0 r=-1->0:-1 component=com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity},
                                                                                                            {null m=TO_BACK f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW p=WCT{RemoteToken{d89069d Task{45cb82d #12070 type=standard A=11008:com.brave.browser}}} leash=Surface(name=ActivityRecord{6e85f0a u0 com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity t12070})/@0xcde9eba sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 component=com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity},
                                                                                                            {WCT{RemoteToken{d89069d Task{45cb82d #12070 type=standard A=11008:com.brave.browser}}} m=CHANGE f=MOVE_TO_TOP leash=Surface(name=Task=12070)/@0xc6d9547 sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 taskParent=-1}
                                                                                                        ]}
2024-05-08 15:53:09.199  2157-2176  WindowManagerShell      com.android.systemui                 V  onTransitionReady (#1665) android.os.BinderProxy@bd9734d: {id=1665 t=OPEN f=0x0 trk=0 r=[0@Point(0, 0)] c=[{null m=OPEN f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW p=WCT{android.window.IWindowContainerToken$Stub$Proxy@e6e8703} leash=Surface(name=ActivityRecord{226fd1 u0 com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity t12070})/@0x6ad9b67 sb=Rect(0, 0 - 0, 0) eb=Rect(0, 0 - 1080, 2400) d=-1->0 r=-1->0:-1 component=com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity},{null m=TO_BACK f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW p=WCT{android.window.IWindowContainerToken$Stub$Proxy@1849680} leash=Surface(name=ActivityRecord{6e85f0a u0 com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity t12070})/@0x5eebd14 sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 component=com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity},{WCT{android.window.IWindowContainerToken$Stub$Proxy@24f44b9} m=CHANGE f=MOVE_TO_TOP leash=Surface(name=Task=12070)/@0x5bac9bd sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 taskParent=-1}]}
2024-05-08 15:53:09.202  9578-9578  cr_A11yState            com.brave.browser                    I  Enabled accessibility services list updated.
2024-05-08 15:53:09.203  9578-9578  cr_A11yState            com.brave.browser                    I  Informing listeners of changes.
2024-05-08 15:53:09.203  9578-9578  cr_A11yState            com.brave.browser                    I  New AccessibilityState: State{isScreenReaderEnabled=false, isTouchExplorationEnabled=false, isPerformGesturesEnabled=false, isAnyAccessibilityServiceEnabled=false, isAccessibilityToolPresent=false, isSpokenFeedbackServicePresent=false, isTextShowPasswordEnabled=true, isOnlyPasswordManagersEnabled=false}
2024-05-08 15:53:09.218  1580-1880  WindowManager           system_server                        V      info={id=1666 t=CLOSE f=0x0 trk=0 r=[0@Point(0, 0)] c=[
                                                                                                            {null m=CLOSE f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW|IS_OCCLUDED leash=Surface(name=ActivityRecord{226fd1 u0 com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity t12070})/@0x9ee3c4b sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 component=com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity},
                                                                                                            {null m=TO_FRONT f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW leash=Surface(name=ActivityRecord{6e85f0a u0 com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity t12070})/@0xcde9eba sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 component=com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity}
                                                                                                        ]}
2024-05-08 15:53:09.218  2157-2176  WindowManagerShell      com.android.systemui                 V  onTransitionReady (#1666) android.os.BinderProxy@61201d6: {id=1666 t=CLOSE f=0x0 trk=0 r=[0@Point(0, 0)] c=[{null m=CLOSE f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW|IS_OCCLUDED leash=Surface(name=ActivityRecord{226fd1 u0 com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity t12070})/@0x3be2c57 sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 component=com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity},{null m=TO_FRONT f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW leash=Surface(name=ActivityRecord{6e85f0a u0 com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity t12070})/@0xbc59344 sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 component=com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity}]}
2024-05-08 15:53:09.218  9578-9578  cr_A11yState            com.brave.browser                    I  Enabled accessibility services list updated.
2024-05-08 15:53:09.219  9578-9578  cr_A11yState            com.brave.browser                    I  Informing listeners of changes.
2024-05-08 15:53:09.219  9578-9578  cr_A11yState            com.brave.browser                    I  New AccessibilityState: State{isScreenReaderEnabled=false, isTouchExplorationEnabled=false, isPerformGesturesEnabled=false, isAnyAccessibilityServiceEnabled=false, isAccessibilityToolPresent=false, isSpokenFeedbackServicePresent=false, isTextShowPasswordEnabled=true, isOnlyPasswordManagersEnabled=false}
2024-05-08 15:53:09.220  2157-2176  WindowManagerShell      com.android.systemui                 V  Transition doesn't have explicit remote, search filters for match for {id=1666 t=CLOSE f=0x0 trk=0 r=[0@Point(0, 0)] c=[{null m=CLOSE f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW|IS_OCCLUDED leash=Surface(name=ActivityRecord{226fd1 u0 com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity t12070})/@0x3be2c57 sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 component=com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity},{null m=TO_FRONT f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW leash=Surface(name=ActivityRecord{6e85f0a u0 com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity t12070})/@0xbc59344 sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 component=com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity}]}
2024-05-08 15:53:09.220  2157-2176  WindowManagerShell      com.android.systemui                 V  start default transition animation, info = {id=1666 t=CLOSE f=0x0 trk=0 r=[0@Point(0, 0)] c=[{null m=CLOSE f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW|IS_OCCLUDED leash=Surface(name=ActivityRecord{226fd1 u0 com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity t12070})/@0x3be2c57 sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 component=com.brave.browser/org.chromium.chrome.browser.sync.ui.PassphraseActivity},{null m=TO_FRONT f=FILLS_TASK|IS_BEHIND_STARTING_WINDOW leash=Surface(name=ActivityRecord{6e85f0a u0 com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity t12070})/@0xbc59344 sb=Rect(0, 0 - 1080, 2400) eb=Rect(0, 0 - 1080, 2400) d=0 component=com.brave.browser/org.chromium.chrome.browser.settings.BraveSettingsActivity}]}
2024-05-08 15:53:09.258  1580-4452  ImeTracker              system_server                        I  com.brave.browser:ae998f4d: onRequestHide at ORIGIN_SERVER reason HIDE_SAME_WINDOW_FOCUSED_WITHOUT_EDITOR fromUser false
2024-05-08 15:53:09.258  1580-4452  ImeTracker              system_server                        I  com.brave.browser:ae998f4d: onCancelled at PHASE_SERVER_SHOULD_HIDE
2024-05-08 15:53:09.260  3253-3253  GoogleInpu...hodService com...gle.android.inputmethod.latin  I  GoogleInputMethodService.onStartInput():1954 onStartInput(EditorInfo{EditorInfo{packageName=com.brave.browser, inputType=0, inputTypeString=NULL, enableLearning=false, autoCorrection=false, autoComplete=false, imeOptions=0, privateImeOptions=null, actionName=UNSPECIFIED, actionLabel=null, initialSelStart=-1, initialSelEnd=-1, initialCapsMode=0, label=null, fieldId=0, fieldName=null, extras=null, hintText=null, hintLocales=[]}}, false)
2024-05-08 15:53:09.261  1580-4452  PackageConfigPersister  system_server                        W  App-specific configuration not found for packageName: com.brave.browser and userId: 0
2024-05-08 15:53:09.736  1580-4452  InputManager-JNI        system_server                        W  Input channel object '7624809 Splash Screen com.brave.browser (client)' was disposed without first being removed with the input manager!

```

## Expected result

`Sync isn't working` notification must not be seen when successfully joining the existing sync chain.

## Issue reproduces how often 

`Easily reproduced`

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? YES
- Can you reproduce this issue with the current Play Store Beta version? YES
- Can you reproduce this issue with the current Play Store Nightly version? YES

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Pixel 6, Pixel 7, Galaxy Tab S8, Galaxy A32 5G
   - Android version: ANY
   
## Brave version

```
Brave build: 1.65.130
Chromium: 124.0.6367.159 (Official Build) (64-bit) 
```

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome? 

### Additional information

cc @AlexeyBarabash

**Comment by NayamAmarshe** [comment]
Same here! Brave sync is unreliable. I've been trying to add a new device to the new chain for hours and it refuses to work!

**Comment by NayamAmarshe** [comment]
I tried the steps in #38101, disabling History sync and it worked! Seems like the sync serve is broken.

**Comment by kjozwiak** [comment]
Closing the above as the fix has been merged into `1.68.x` & `1.67.x` as per https://github.com/brave/brave-core/pull/24239 & https://github.com/brave/brave-core/pull/24238. Also labelling as `QA/No` & `release-notes/exclude` in favor of https://github.com/brave/brave-browser/issues/38847.
